Sources/UnidirectionalBinding.swift

```diff
@@ -276,3 +277,62 @@ extension BindingTarget where Value: OptionalProtocol {
 		return target <~ property.producer
 	}
 }
+
+/// A pseudo `BindingTarget`, constructed from a closure and a lifetime.
+public final class AnyBindingTarget<Value>: BindingTarget {
```

Comment:
Have we considered any other names than `AnyBindingTarget`? I can see from the initialiser below that this can be used as a type-erased wrapper, but I'm not sure that's its primary purpose. Given that this is perfectly useful as just a pure sink for a binding, what about calling it `BindingSink` or `Sink` or even just `BindingTarget` (the latter implying a rename of the protocol to `BindingTargetProtocol`)?

I'm not necessarily opposed to the name, just thought I'd throw some alternatives out there :)